assets/js/blocks/cart/inner-blocks/cart-cross-sells-products/block.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { useStoreCart } from '@woocommerce/base-context/hooks';
+
+/**
+ * Internal dependencies
+ */
+import CartCrossSellsProductList from '../../cart-cross-sells-product-list';
+import metadata from './block.json';
+
+interface BlockProps {
+	className?: string;
+	columns: number;
+}
+
+const Block = ( { className, columns }: BlockProps ): JSX.Element => {
+	const { crossSellsProducts, cartIsLoading } = useStoreCart();
+
+	if ( typeof columns === 'undefined' ) {
+		columns = metadata.attributes.columns.default;
```

Comment:
Are we guaranteed to always have this, would `metadata?.attributes?.columns?.default` be safer or is it unnecessary?

Reply:
> Are we guaranteed to always have this [...]?

Based on the following section in block.json, I was under the impression that `metadata.attributes.columns.default` always exists:

```json
"attributes": {
  "columns": {
    "type": "number",
    "default": 3
  }
},
```

Am I missing something in this case?

Reply:
What I meant was: is there any case where `metadata` or any of its children could be undefined, or a non-object. Looking at the rest of the codebase for `metadata.attributes`, it seems we're not worried about that. Marking as resolved!